Unexpeted type tag 13996 found.

When adding the gradle plugin I´m getting this error (the tag number changes every time)
Any idea?
### Project Setup
`build.gradle`
```Gradle
apply from: 'buildsystem/ci.gradle'
apply from: 'buildsystem/dependencies.gradle'

buildscript {
    repositories {
        jcenter()
        mavenCentral()
        google()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.0.1'
        classpath 'com.jakewharton:butterknife-gradle-plugin:8.4.0'
        classpath 'me.tatarka:gradle-retrolambda:3.6.0'
        classpath "io.realm:realm-gradle-plugin:3.5.0"
        classpath 'com.google.gms:google-services:3.1.2'
    }
}
allprojects {
    repositories {
        maven {
            url "https://jitpack.io"
        }
        maven {
            url "https://maven.google.com"
        }
        google()
    }
    ext {
        androidApplicationId = 'com.iagt.android.opeople.presentation'
        androidVersionCode = 1
        androidVersionName = "1.0"
        testInstrumentationRunner = "android.support.test.runner.AndroidJUnitRunner"
        testApplicationId = 'com.innopro.android.sample.presentation.test'
    }
}

task wrapper(type: Wrapper) {
    description 'Creates the gradle wrapper.'
    gradleVersion '4.1'
}

task runDomainUnitTests(dependsOn: [':domain:test']) {
    description 'Run unit tests for the domain layer.'
}

task runDataUnitTests(dependsOn: [':data:cleanTestDebugUnitTest', ':data:testDebugUnitTest']) {
    description 'Run unit tests for the data layer.'
}

task runUnitTests(dependsOn: ['runDomainUnitTests', 'runDataUnitTests']) {
    description 'Run unit tests for both domain and data layers.'
}

task runAcceptanceTests(dependsOn: [':presentation:connectedAndroidTest']) {
    description 'Run application acceptance tests.'
}
```


`presentation/build.gradle`
```Gradle
plugins {
    id 'com.onesignal.androidsdk.onesignal-gradle-plugin' version '0.8.0'
}
apply plugin: 'com.onesignal.androidsdk.onesignal-gradle-plugin'

repositories {
    maven { url 'https://maven.google.com' }
}
apply plugin: 'com.google.gms.google-services'
apply plugin: 'com.android.application'
apply plugin: 'com.jakewharton.butterknife'
configurations {
    iagtprodDebugCompile
}
android {
    def globalConfiguration = rootProject.extensions.getByName("ext")

    compileSdkVersion globalConfiguration.getAt("androidCompileSdkVersion")
    buildToolsVersion globalConfiguration.getAt("androidBuildToolsVersion")

    defaultConfig {
        minSdkVersion globalConfiguration.getAt("androidMinSdkVersion")
        targetSdkVersion globalConfiguration.getAt("androidTargetSdkVersion")

        applicationId globalConfiguration.getAt("androidApplicationId")
        versionCode globalConfiguration.getAt("androidVersionCode")
        versionName globalConfiguration.getAt("androidVersionName")
        testInstrumentationRunner globalConfiguration.getAt("testInstrumentationRunner")
        testApplicationId globalConfiguration.getAt("testApplicationId")
        multiDexEnabled true
        manifestPlaceholders = [onesignal_app_id: "XXXXXXXXXXXXXXXXXXXXXXXX",
                                onesignal_google_project_number: "REMOTE"]

    }

    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }

    packagingOptions {
        exclude 'LICENSE.txt'
        exclude 'META-INF/DEPENDENCIES'
        exclude 'META-INF/ASL2.0'
        exclude 'META-INF/NOTICE'
        exclude 'META-INF/LICENSE'
        exclude 'META-INF/rxjava.properties'
    }

    lintOptions {
        quiet true
        abortOnError false
        ignoreWarnings true
        disable 'InvalidPackage'            //Some libraries have issues with this.
        disable 'OldTargetApi'
        //Lint gives this warning but SDK 20 would be Android L Beta.
        disable 'IconDensities'             //For testing purpose. This is safe to remove.
        disable 'IconMissingDensityFolder'  //For testing purpose. This is safe to remove.
    }

    signingConfigs {
        debug {
            storeFile file('../buildsystem/debug.keystore')
            storePassword 'XXXXXX'
            keyAlias 'XXXXXX'
            keyPassword 'XXXXX'
        }
    }

    buildTypes {
        debug {
            signingConfig signingConfigs.debug
        }
    }
    flavorDimensions "default"
    productFlavors {
        iagtprod {
            dimension "default"
            buildConfigField "boolean","prodVersion", "false"
            resValue "string", "facebook_app_id","\"XXXXXXXXXXXXXXX\""
            resValue "string", "facebook_login_protocol_scheme","\"fbXXXXXXXXXXXX\""
        }
        prod{
            dimension "default"
            buildConfigField "boolean","prodVersion", "true"
            resValue "string", "facebook_app_id","\"XXXXXXXXXXXXXXX\""
            resValue "string", "facebook_login_protocol_scheme","\"fbXXXXXXXXXXXX\""
        }
    }
}

dependencies {
    def presentationDependencies = rootProject.ext.presentationDependencies
    def presentationTestDependencies = rootProject.ext.presentationTestDependencies

    annotationProcessor presentationDependencies.daggerCompiler
    annotationProcessor presentationDependencies.butterKnifeCompiler
    annotationProcessor presentationDependencies.fragmentArgsPro
    compile presentationDependencies.googleMap
    compile presentationDependencies.googleMapUtils
    compile presentationDependencies.googleLocation
    compile presentationDependencies.googlePlaces
    compile presentationDependencies.googleAuth
    compile presentationDependencies.firebaseCore
    compile presentationDependencies.firebaseAuth
    compile presentationDependencies.dagger
    compile presentationDependencies.butterKnife
    compile presentationDependencies.recyclerView
    compile presentationDependencies.rxJava
    compile presentationDependencies.rxAndroid
    compile presentationDependencies.picasso
    compile presentationDependencies.appCompat
    compile presentationDependencies.supportDesign
    compile presentationDependencies.transformations
    compile presentationDependencies.androidAnnotations
    compile presentationDependencies.fragmentArgs
    compile presentationDependencies.hideKeyboard
    compile presentationDependencies.joda
    compile presentationDependencies.pulsator
    compile presentationDependencies.calligraphy
    compile presentationDependencies.passwordedEdittext
    compile presentationDependencies.stickyList
    compile presentationDependencies.multiCollapsing
    compile presentationDependencies.facebook
    compile presentationDependencies.multiDex


    implementation project(path: ':domain')
    implementation project(path: ':data')
    /*prodCompile project(path: ':domain')
    prodCompile project(path: ':data')*/
/*
  androidTestCompile presentationTestDependencies.mockito
  androidTestCompile presentationTestDependencies.espresso
  androidTestCompile presentationTestDependencies.androidAnnotations*/

    compile 'fr.avianey.com.viewpagerindicator:library:2.4.1.1@aar'
    compile 'com.android.support:support-v4:25.4.0'
    compile 'com.android.support.constraint:constraint-layout:1.0.2'
    compile 'com.onesignal:OneSignal:3.7.1'
}
```

### Full Error
```Java
Error:Unexpected type tag 13996 found.
```

### Dependency Tree
```
FAILURE: Build failed with an exception.

* What went wrong:
Project 'app' not found in root project 'opeople'.

* Try:
Run with --info or --debug option to get more log output.

* Exception is:
org.gradle.execution.taskpath.ProjectFinderByTaskPath$ProjectLookupException: Project 'app' not found in root project 'opeople'.
        at org.gradle.execution.taskpath.ProjectFinderByTaskPath.findProject(ProjectFinderByTaskPath.java:47)
        at org.gradle.execution.taskpath.TaskPathResolver.resolvePath(TaskPathResolver.java:49)
        at org.gradle.execution.TaskSelector.getSelection(TaskSelector.java:95)
        at org.gradle.execution.TaskSelector.getSelection(TaskSelector.java:81)
        at org.gradle.execution.commandline.CommandLineTaskParser.parseTasks(CommandLineTaskParser.java:42)
        at org.gradle.execution.TaskNameResolvingBuildConfigurationAction.configure(TaskNameResolvingBuildConfigurationAction.java:44)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter.configure(DefaultBuildConfigurationActionExecuter.java:48)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter.access$000(DefaultBuildConfigurationActionExecuter.java:25)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter$1.proceed(DefaultBuildConfigurationActionExecuter.java:54)
        at org.gradle.execution.DefaultTasksBuildExecutionAction.configure(DefaultTasksBuildExecutionAction.java:44)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter.configure(DefaultBuildConfigurationActionExecuter.java:48)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter.access$000(DefaultBuildConfigurationActionExecuter.java:25)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter$1.proceed(DefaultBuildConfigurationActionExecuter.java:54)
        at org.gradle.execution.ExcludedTaskFilteringBuildConfigurationAction.configure(ExcludedTaskFilteringBuildConfigurationAction.java:47)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter.configure(DefaultBuildConfigurationActionExecuter.java:48)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter.select(DefaultBuildConfigurationActionExecuter.java:36)
        at org.gradle.initialization.DefaultGradleLauncher$CalculateTaskGraph.run(DefaultGradleLauncher.java:268)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
        at org.gradle.initialization.DefaultGradleLauncher.constructTaskGraph(DefaultGradleLauncher.java:175)
        at org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:130)
        at org.gradle.initialization.DefaultGradleLauncher.executeTasks(DefaultGradleLauncher.java:109)
        at org.gradle.internal.invocation.GradleBuildController$1.call(GradleBuildController.java:78)
        at org.gradle.internal.invocation.GradleBuildController$1.call(GradleBuildController.java:75)
        at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:152)
        at org.gradle.internal.invocation.GradleBuildController.doBuild(GradleBuildController.java:100)
        at org.gradle.internal.invocation.GradleBuildController.run(GradleBuildController.java:75)
        at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:28)
        at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
        at org.gradle.tooling.internal.provider.ValidatingBuildActionRunner.run(ValidatingBuildActionRunner.java:32)
        at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner$1.run(RunAsBuildOperationBuildActionRunner.java:43)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
        at org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner.run(RunAsBuildOperationBuildActionRunner.java:40)
        at org.gradle.tooling.internal.provider.SubscribableBuildActionRunner.run(SubscribableBuildActionRunner.java:51)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:47)
        at org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:30)
        at org.gradle.launcher.exec.BuildTreeScopeBuildActionExecuter.execute(BuildTreeScopeBuildActionExecuter.java:39)
        at org.gradle.launcher.exec.BuildTreeScopeBuildActionExecuter.execute(BuildTreeScopeBuildActionExecuter.java:25)
        at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:80)
        at org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:53)
        at org.gradle.tooling.internal.provider.ServicesSetupBuildActionExecuter.execute(ServicesSetupBuildActionExecuter.java:57)
        at org.gradle.tooling.internal.provider.ServicesSetupBuildActionExecuter.execute(ServicesSetupBuildActionExecuter.java:32)
        at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:36)
        at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:25)
        at org.gradle.tooling.internal.provider.ParallelismConfigurationBuildActionExecuter.execute(ParallelismConfigurationBuildActionExecuter.java:43)
        at org.gradle.tooling.internal.provider.ParallelismConfigurationBuildActionExecuter.execute(ParallelismConfigurationBuildActionExecuter.java:29)
        at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:64)
        at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:29)
        at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:59)
        at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:44)
        at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:45)
        at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:30)
        at org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:67)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:122)
        at org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:37)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:122)
        at org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:26)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:122)
        at org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDaemon.java:34)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:122)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.call(ForwardClientInput.java:74)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.call(ForwardClientInput.java:72)
        at org.gradle.util.Swapper.swap(Swapper.java:38)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:72)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:122)
        at org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:55)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:122)
        at org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:62)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:122)
        at org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.java:82)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:36)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:122)
        at org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.java:50)
        at org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:295)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
        at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)

```

--- a/src/main/groovy/com/onesignal/androidsdk/GradleProjectPlugin.groovy
+++ b/src/main/groovy/com/onesignal/androidsdk/GradleProjectPlugin.groovy
@@ -80,12 +80,14 @@ class GradleProjectPlugin implements Plugin<Project> {
     static def versionGroupAligns
     static Project project
     static def moduleVersionOverrides
+    static def moduleCopied
 
     @Override
     void apply(Project inProject) {
         project = inProject
         versionGroupAligns = InternalUtils.deepcopy(VERSION_GROUP_ALIGNS)
         moduleVersionOverrides = [:]
+        moduleCopied = [:]
 
         resolutionHooksForAndroidPluginV3()
         resolutionHooksForAndroidPluginV2()
@@ -99,7 +101,6 @@ class GradleProjectPlugin implements Plugin<Project> {
                     return
 
                 def configuration = variant.compileConfiguration
-
                 doResolutionStrategyAndroidPluginV3(configuration)
             }
         }
@@ -108,10 +109,10 @@ class GradleProjectPlugin implements Plugin<Project> {
     static void resolutionHooksForAndroidPluginV2() {
         project.configurations.all { configuration ->
             project.afterEvaluate {
-                if (!isAndroidPluginV3()) {
-                    generateHighestVersionsForGroups(configuration)
-                    doResolutionStrategyAndroidPluginV2(configuration)
-                }
+                if (isAndroidPluginV3())
+                    return
+                generateHighestVersionsForGroups(configuration)
+                doResolutionStrategyAndroidPluginV2(configuration)
             }
 
             // Catches Android specific tasks, <buildType>CompileClasspath
@@ -165,7 +166,7 @@ class GradleProjectPlugin implements Plugin<Project> {
             targetSdkVersion = mergedFlavor.targetSdkVersion.apiLevel
         }
 
-        return targetSdkVersion
+        targetSdkVersion
     }
 
     static void doTargetSdkVersionAlign(DependencyResolveDetails details) {
@@ -238,7 +239,7 @@ class GradleProjectPlugin implements Plugin<Project> {
         project.logger.info("OneSignalProjectPlugin: ${msg}")
     }
 
-    static boolean inGroupAlignList(def details) {
+    static boolean inGroupAlignList(details) {
         // Only override groups we define
         def versionOverride = versionGroupAligns[details.requested.group]
         if (!versionOverride)
@@ -248,16 +249,20 @@ class GradleProjectPlugin implements Plugin<Project> {
         def omitModules = versionOverride['omitModules']
         if (omitModules && omitModules.contains(details.requested.name))
             return false
-        return true
+        true
     }
 
     static String getHighestVersion(String version1, String version2) {
-        return compareVersions(version1, version2) > 0 ? version1 : version2
+        compareVersions(version1, version2) > 0 ? version1 : version2
+    }
+
+    static String moduleGroupAndName(details) {
+        "${details.requested.group}:${details.requested.name}"
     }
 
     static int compareVersions(String version1, String version2) {
         def versionComparator = new DefaultVersionComparator()
-        return versionComparator.compare(new VersionInfo(version1), new VersionInfo(version2))
+        versionComparator.compare(new VersionInfo(version1), new VersionInfo(version2))
     }
 
     static Object finalAlignmentRules() {
@@ -268,7 +273,7 @@ class GradleProjectPlugin implements Plugin<Project> {
         updateMockVersionsIntoGradleVersions(finalVersionGroupAligns)
 
         project.logger.debug("OneSignalProjectPlugin: FINAL ALIGN PART 2: ${finalVersionGroupAligns}")
-        return finalVersionGroupAligns
+        finalVersionGroupAligns
     }
 
     static void alignAcrossGroups(def versionGroupAligns) {
@@ -298,14 +303,24 @@ class GradleProjectPlugin implements Plugin<Project> {
     // project.android.@plugin - This looks to be on the AppExtension class however this didn't work
     // Found 'enforceUniquePackageName' by comparing project.android.properties between versions
     static boolean isAndroidPluginV3() {
-        return !project.android.hasProperty('enforceUniquePackageName')
+        !project.android.hasProperty('enforceUniquePackageName')
+    }
+
+    static void forceCanBeResolved(def configuration) {
+        // canBeResolved not available on Gradle 2.14.1 and older
+        if (configuration.hasProperty('canBeResolved'))
+            configuration.canBeResolved = true
     }
 
     static void generateHighestVersionsForGroups(def configuration) {
+        // Prevent duplicate runs for the same configuration name
+        //   Fixes infinite calls when multiDexEnabled is set
+        if (moduleCopied[configuration.name])
+            return
+        moduleCopied[configuration.name] = true
+
         def configCopy = configuration.copy()
-        // canBeResolved not available on Gradle 2.14.1 and older
-        if (configCopy.hasProperty('canBeResolved'))
-            configCopy.canBeResolved = true
+        forceCanBeResolved(configCopy)
 
         configCopy.resolutionStrategy.eachDependency { DependencyResolveDetails details ->
             doTargetSdkVersionAlign(details)
@@ -329,7 +344,7 @@ class GradleProjectPlugin implements Plugin<Project> {
         // Checking for configuration.name == 'compile' || 'implementation' skips to much however
         try {
            configuration.resolvedConfiguration.resolvedArtifacts
-        } catch (def e) {}
+        } catch (any) {}
     }
 
     static String getVersionFromDependencyResolveDetails(DependencyResolveDetails details) {
@@ -347,8 +362,6 @@ class GradleProjectPlugin implements Plugin<Project> {
             return '9999.9999.9999'
         else
             project.logger.error("OneSignalProjectPlugin: Unkown VersionSelector: ${parsedVersion}")
-
-        return null
     }
 
     static final int MAJOR_INDEX = 0
@@ -381,6 +394,6 @@ class GradleProjectPlugin implements Plugin<Project> {
                 parts[MINOR_INDEX] = minorVersion.toString()
         }
 
-        return parts.join('.')
+        parts.join('.')
     }
 }

--- a/src/test/groovy/com/onesignal/androidsdk/MainTest.groovy
+++ b/src/test/groovy/com/onesignal/androidsdk/MainTest.groovy
@@ -91,6 +91,7 @@ class MainTest extends Specification {
                     targetSdkVersion ${buildSections['targetSdkVersion']}
                     versionCode 1
                     versionName "1.0"
+                    multiDexEnabled true
                 }
                 
                 buildTypes {
@@ -204,7 +205,7 @@ class MainTest extends Specification {
         then:
         assert results // Asserting existence and contains 1+ entries
         results.each {
-            assert it.value.contains('\\--- com.onesignal:OneSignal:3.5.+ -> 3.6.3')
+            assert it.value.contains('--- com.onesignal:OneSignal:3.5.+ -> 3.6.3')
             assert it.value.contains(' +--- com.google.android.gms:play-services-gcm:[10.2.1,11.3.0) -> 11.2.2')
         }
     }
@@ -233,7 +234,7 @@ class MainTest extends Specification {
 
         then:
         results.each {
-            assert it.value.contains('\\--- com.android.support:support-v4:26.0.0 -> 25.4.0')
+            assert it.value.contains('--- com.android.support:support-v4:26.0.0 -> 25.4.0')
         }
     }
 
@@ -251,7 +252,7 @@ class MainTest extends Specification {
         results.each {
             assert it.value.contains('+--- com.google.firebase:firebase-core:11.0.0 -> 11.4.0')
             assert it.value.contains('+--- com.google.android.gms:play-services-gcm:11.2.0 -> 11.4.0')
-            assert it.value.contains('\\--- com.google.android.gms:play-services-location:11.4.0')
+            assert it.value.contains('--- com.google.android.gms:play-services-location:11.4.0')
         }
     }
 
@@ -282,7 +283,7 @@ class MainTest extends Specification {
         then:
         results.each {
             assert it.value.contains('+--- com.android.support:appcompat-v7:25.0.+ -> 25.4.0')
-            assert it.value.contains('\\--- com.android.support:support-v4:25.+ -> 25.4.0 (*)')
+            assert it.value.contains('--- com.android.support:support-v4:25.+ -> 25.4.0 (*)')
         }
     }
 
@@ -298,7 +299,7 @@ class MainTest extends Specification {
         then:
         results.each {
             assert it.value.contains('+--- com.android.support:appcompat-v7:[25.0.0, 26.0.0) -> 26.0.0-beta2')
-            assert it.value.contains('\\--- com.android.support:support-v4:25.+ -> 26.0.0-beta2 (*)')
+            assert it.value.contains('--- com.android.support:support-v4:25.+ -> 26.0.0-beta2 (*)')
         }
     }
 
@@ -386,8 +387,8 @@ class MainTest extends Specification {
             assert it.value.contains('+--- com.google.android.gms:play-services-gcm:[10.2.1,11.3.0) -> 11.2.2')
             assert it.value.contains('+--- com.google.android.gms:play-services-location:[10.2.1,11.3.0) -> 11.2.2')
             assert it.value.contains('+--- com.android.support:support-v4:[26.0.0,26.2.0) -> 26.1.0 (*)')
-            assert it.value.contains('\\--- com.android.support:appcompat-v7:25.0.0 -> 26.1.0')
-            assert it.value.contains('\\--- com.android.support:customtabs:[26.0.0,26.2.0) -> 26.1.0')
+            assert it.value.contains('--- com.android.support:appcompat-v7:25.0.0 -> 26.1.0')
+            assert it.value.contains('--- com.android.support:customtabs:[26.0.0,26.2.0) -> 26.1.0')
         }
     }
 }

